src/main.rs

```diff
@@ -119,3 +132,17 @@ fn main() {
     }
 
 }
+
+fn discover_codeowners() -> Option<PathBuf> {
+    let curr_dir = current_dir().unwrap();
```

Comment:
If possible could we avoid unwrapping?

Reply:
Sure thing. Docs say that `current_dir()` could be `Err` if (perhaps not comprehensive?)
* Current directory does not exist.
* There are insufficient permissions to access the current directory.

Perhaps desired behavior could be printing something like "Could not confirm working directory. Does it still exist? Is it readable?", then exit 2/some not-yet-taken number. What do you think?